frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-treatment-status/radiogram-card-content-treatment-status.component.html

```diff
@@ -0,0 +1,24 @@
+<ng-container>
```

Comment:
What are those for?

Reply:
They behave as elements in the template but do not create elements in HTML. This allows us to use directives more freely. For instance, an `*ngSwitchCase` as `<ng-container>` can include multiple children that are rendered without any surrounding `<div>`.